docs/configuration/sendrecvoptions.rst

```diff
@@ -79,6 +79,29 @@ If ``encrypted=false``, zrepl expects that filesystems matching ``filesystems``
    Use ``encrypted`` instead of ``raw`` to make your intent clear that zrepl must only replicate filesystems that are actually encrypted by OpenZFS native encryption.
    It is meant as a safeguard to prevent unintended sends of unencrypted filesystems in raw mode.
 
+.. _job-send-options-properties:
+
+``properties``
+--------------
+Sends the dataset properties along with snapshots.
+
+.. NOTE::
+
+   Please read the :ref:`note on property replication below <job-note-property-replication>`.
+
+.. _job-send-options-backup-properties:
+
+``backup_properties``
+---------------------
+
+When properties are modified on a dataset that was received from a send stream with properties, zfs archives the original received value internally.
+This also applies to inheriting or overriding properties during receive.
+
+This flag instructs zfs send to send those original property values rather than the current locally set values.
+
+This is useful for replicating properties from first level to second level backups, on which you'd want to locally override at least ``mountpoint`` or ``canmount`` (see the :ref:`note on property replication below <job-note-property-replication>`) but don't want to lose the original value for restore purposes.
+
+Enabling this flag on the first level backup machine will cause it to send the values originally received from the backup source to the second level backup, which can perform the necessary property overrides/inherits locally for itself again. This setup can be chained infinitely.
```

Comment:
about this whole section: It is essentially a rough outline of how one would approach multi-tier replication with zrepl. Is it a good idea to bury it within the description of send option? Shouldn't it be in the Use Case / Tutorial section?

Reply:
I think trying to cram every bit of knowledge like this into a use case/tutorial section will overflow it. This is advice on how to handle receiving properties in the section about enabling sending and receiving properties. Seems reasonable to me.

I would expect the tutorial sections to make heavy reference to it though.

Reply:
Oh, derp, rereading it, it is very tutorial-ly. My bad.
We should probably move and expand it and then reference it here, I guess.

Reply:
Maybe push it down to the `job-note-property-replication` section? It already contains a bunch of prose...